scheduler/remote_server.py

```diff
@@ -77,6 +91,8 @@ def plot_timeline(timeline_list, timeline_dict, scd_dir, now):
 
     first_date, last_date = None, None
 
+    timeline_list = [0] * len(timeline)
```

Comment:
When I left it as overwriting the object output from scd_to_timeline it broke timeline_to_atq because it didn't have the fields referenced when converting it to a dictionary for that function. 

I didn't want to mess around with how the actual scheduling was being done so I had it create the new timeline_list for plotting purposes.